problemsets-mat223/modules/module2.tex

```diff
@@ -186,8 +186,8 @@
 \]
 corresponds to the line passing through $\mat{p_1\\p_2}$ with $\mat{d_1\\d_2}$ as a direction vector.
 
-The ``$t$'' that appears in a vector form is called the \emph{parameter variable}, and for this
-reason, some textbooks use the term \emph{parametric form} in place of ``vector form''. 
+The ``$t$'' that appears in a vector form is called the \emph{parameter variable}\index[definitions]{line!parameter variable}\index{line!parameter variable}, and for this
+reason, some textbooks use the term \emph{parametric form}\index{line!parametric form}\index[definitions]{line!parametric form} in place of ``vector form''. 
```

Comment:
It is stated `some textbooks use`... as an indication that *this textbook does **not** use*. As a consequence, this doesn't belong in the `definitions` index (it can go in the regular index, though).